config/overlays/tls/kustomization.yaml

```diff
@@ -0,0 +1,27 @@
+# Adds namespace to all resources.
+namespace: operator-controller-system
+
+# Value of this field is prepended to the
+# names of all resources, e.g. a deployment named
+# "wordpress" becomes "alices-wordpress".
+# Note that it should also match with the prefix (text before '-') of the namespace
+# field above.
+namePrefix: operator-controller-
+
+# the following config is for teaching kustomize how to do var substitution
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../../base/crd
+- ../../base/rbac
+- ../../base/manager
```

Comment:
If we are able to drop the `./config/base/default` kustomization, we could add a `./config/base` kustomization and then here just use `../../base`.